src/generate-names.h

```diff
@@ -26,7 +26,8 @@ struct Module;
 enum NameOpts {
```

Comment:
Nit: if these are always fully qualified, would we want to make this an enum class to enforce that? (and get things out of the global namespace)

Reply:
`enum class` is a pain with bit flags, as you need to define a custom `operator|` with lots of casts to make it work.